### DISCUSSION TITLE
Extensões & Add-ons

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#85] [Perguntas e Respostas]
Pessoal gostaria de uma troca de informações sobre extensões úteis na hora do desenvolvimento, no meu caso o dev tools do Chrome faz quase tudo, fora ele Colorzilla e o LiveReload, me mostraram uma extensão chamada Wappalyzer hoje também que mostra as tecnologias utilizada no site em que você está, achei bem maneira.

**Comment by rickbenetti** [comment]
Como estou mudando de área hoje mantenho estes:
- foco front: wappalyzer, awesome screenshot, gist box, colorzilla, toggl timer,  github notifier.
- foco design: wappalyzer, awesome screenshot, screencastify, colorzilla, toogl timer.

outros que tenho: readability (no safari é nativo), pinterest, evernote, dropbox, convert to markdown.

**Comment by lucasbento** [comment]
- [Wappalyzer](https://chrome.google.com/webstore/detail/wappalyzer/gppongmhjkpfnbhagpmjfkannfbllamg): veja a stack que a página está utilizando;
- [Markdown Here](http://markdown-here.com/): escreva qualquer coisa em um campo WYSIWYG usando markdown (não é bem segmentado pra dev mas ajuda pra caralho);
- [JSONView](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc): JSON formatado;
- [Octotree](https://chrome.google.com/webstore/detail/octotree/bkhaagjahfmjljalopjnoealnfndnagc): Arquivos dos projetos no github exibidos em árvore;
- [Colorzilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp): colorpicker;
- [Postman](https://chrome.google.com/webstore/detail/postman/fhbjgbiflinjbdggehcddcbncdddomop): Testar suas APIs REST;
- [Firebug (Firefox)](https://addons.mozilla.org/pt-br/firefox/addon/firebug/): eu não sei porquê mas eu não gosto do Chrome DevTools :/

Não conhecia o LiveReload, vou testar!

**Comment by felipefialho** [comment]
Para foco:
- [Tomato Timer](http://tomato-timer.com/#) - Para contar pomodoros

Extensões:
- DevTools - Básico
- [Colorzilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp) - Colorzilla... padrão
- [Page Ruler](https://chrome.google.com/webstore/detail/page-ruler/jlpkojjdgbllmedoapgfodplfhcbnbpn?hl=en) - Uma régua maneira para medir alinhamentos
- [TechSmith Snagit](https://chrome.google.com/webstore/detail/techsmith-snagit-extensio/annopcfmbiofommjmcmcfmhklhgbhkce?hl=en) - O melhor que testei para gravar videos e pegar screen das telas

---

Já usei muito mais coisas, mas atualmente só isso da conta do recado.

**Comment by mksanti** [comment]
Meu navegador padrão é o FF e nele o devtools ja tem um Colorzilla embutido + page rules + page screenshot. Fora isso para complementar utilizo o Evernote Web clipper para salvar os artigos legais da net e wappalyzer.

@rkb81 Gostei de alguns add-ons que vc utiliza vou testar aqui tb.

**Comment by DevCarlosAugusto** [comment]
Vou testar essa 'TechSmith Snagit', 'Tomato Timer' e 'Octotree'.
Já uso o JSONView, colorzilla, the great suspender, editthiscookie e gridruler.

**Comment by vitor-mendrone-deel** [comment]
Por aqui eu utilizo: 
- [Save To Pocket](https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj) (para salvar artigos interessantes)
- [PerfectPixel](https://chrome.google.com/webstore/detail/perfectpixel-by-welldonec/dkaagdgjmgdmbnecmcefdhjekcoceebi) (cria uma layer com transparência pra comparar uma imagem com o site)
- [Google Tag Assistant](https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk) (Como trampo com E-commerce, ajuda a saber quais tags (Analytics e Adwords) a loja utiliza e se foram implementadas corretamente)
- [SEO Stats](https://chrome.google.com/webstore/detail/open-seo-statsformerly-pa/hbdkkfheckcdppiaiabobmennhijkknn) - Faz uma análise rápida do site com diversas outras informações úteis (número de páginas indexadas em diversos sites de busca, pagerank, etc)
- [Page Ruler](https://chrome.google.com/webstore/detail/page-ruler/jlpkojjdgbllmedoapgfodplfhcbnbpn) (Régua pra verificar alinhamentos)
- [Emmet re:view](https://chrome.google.com/webstore/detail/emmet-review/epejoicbhllgiimigokgjdoijnpaphdp) (Mostra múltiplas versões da página em uma variedade de viewports *by Emmet Team)

**Comment by felipefialho** [comment]
Legal esse Emmet re:view, não conhecia.

**Comment by dougaraujos** [comment]
Além dos citados anteriormente: 
- [Dimensions](http://felixniklas.com/dimensions/) (para mensurar elementos e imagens no browser)
- [SVG Navigator](https://chrome.google.com/webstore/detail/svg-navigator/pefngfjmidahdaahgehodmfodhhhofkl) (abrir arquivos SVG no Chrome)

**Comment by tcelestino** [comment]
Mesmo achando o DevTools do Chrome muito mais completo, uso o DevTools do Firefox e Firefox Developer para "debugar" as aplicações que estou desenvolvendo, além das extensões abaixo:
- [GreaskeMonkey](http://www.greasespot.net/) - se curte customizar, alterar, remover etc... algo em algum site, com essa extensão fica bem simples de realizar isso.
- [Forest](http://www.forestapp.cc/) - para vc não sair do foco. Essa comecei usar recentemente
- [JSONView](http://jsonview.com/) - deixe a visualização de dados via JSON mais "pretty".
- [RestClient](http://restclient.net/) - alternativa do POSTman do Chrome para o Firefox
- [React DevTools](https://github.com/facebook/react-devtools) - melhor ferramenta para debugar seus componentes React.
- [Backbone Eye](http://dhruvaray.github.io/spa-eye/) - debugar aplicações em Backbone. Só ativo quando uso o Firebug (só por causa dele)
- [User Agent Override](https://github.com/muzuiget/user_agent_overrider) - o Firefox ainda não tem um gerenciador de user-agent nativo (parece que logo, logo, teremos), como o Chrome ou o Safari.

**Extras**
- [Ghostery](https://www.ghostery.com/) - desativa um monte de scripts que rodam por trás quando você acessa um site.
- [GSantizer](https://github.com/shadyabhi/GSanitizer) - remove o tracking do Google quando clica em algum link no resultado da busca.
- [LastPass](https://lastpass.com/) - gerenciador de senhas
- [Pushbullet](https://www.pushbullet.com/) - compartilhe tudo em diversos dispositivos
- [Hide Unwanted Results of Google Search](https://addons.mozilla.org/pt-BR/firefox/addon/hide-unwanted-results-of-go/?src=api) - remove resultados de busca no Google que você não quer como referência. Eu uso por exemplo para esconder todos os links do W3Schools

**Comment by victorserpac** [comment]
To meio atrasado na Issue, mas eu gosto de usar o [Fontface Ninja](http://www.fontface.ninja) pra debugar fontes em geral.

**Comment by fabricionaweb** [comment]
@tcelestino testando esse **forest** caiu como uma luva, achei genial e estava buscando um pomodoro motivacional 😁
